Documentation/Configuration/BackendEntryPoint/Index.rst

```diff
@@ -136,7 +111,7 @@ Apache configuration
 
 It is most important to rewrite all `typo3/*` requests to `/index.php`, but also
 `RewriteCond %{REQUEST_FILENAME} !-d` should be removed in order for a request
-to `/typo3/` to be directly served via `/index.php` instead of the deprecated
+to `/typo3/` to be directly served via `/index.php` instead of the removed
 entry point `/typo3/index.php`.
```

Comment:
The sentence is too long and incomprehensible to me. Can you separate the long sentences in at least two pieces?

Reply:
The sentence was already there, I changed only the one word. The rest would be for a follow up